randovania/games/dread/exporter/patch_data_factory.py

```diff
@@ -376,6 +376,14 @@ def _cosmetic_patch_data(self) -> dict:
                 },
                 "camera_names_dict": self._build_area_name_dict()
             },
+            "shield_versions": {
+                "ice_missile": ("ALTERNATE" if c.alt_ice_missile else "DEFAULT"),
+                "storm_missile": ("ALTERNATE" if c.alt_storm_missile else "DEFAULT"),
+                "diffusion_beam": ("ALTERNATE" if c.alt_diffusion_beam else "DEFAULT"),
+                "bomb": ("ALTERNATE" if c.alt_bomb else "DEFAULT"),
+                "cross_bomb": ("ALTERNATE" if c.alt_cross_bomb else "DEFAULT"),
+                "power_bomb": ("ALTERNATE" if c.alt_power_bomb else "DEFAULT"),
```

Comment:
these should all be enums instead of bools, with values of "ALTERNATE" and "DEFAULT" so that you can get rid of this ugly ternary